tech-cent/momo_bank_web
# #170670280 Retrieve transactions

**What does this PR do?**
This PR adds functionality to retrieve account transactions once a user is logged in.

**Description of the task to be completed?**
- design the dashboard with a main and side panel
- implement fetching of transactions

**How should this be manually tested?**
- Clone and set up the project using the instruction in the README.
- $ git checkout to `ft-view-transactions-170670280` branch which contains the changes.
- Use $ npm start to start the application
- Login to the momo bank web app
You should see transactions made on your default account.

**What are the relevant pivotal tracker stories?**
[#170670280](https://www.pivotaltracker.com/story/show/170670280)

**Screenshots**
<img width="1231" alt="Screenshot 2020-01-15 at 20 04 09" src="https://user-images.githubusercontent.com/38077368/72454606-392afd80-37d2-11ea-9f5f-4216c7f4e12a.png">
